internal/report/report.go

```diff
@@ -264,6 +264,10 @@ func (rpt *Report) newGraph(nodes graph.NodeSet) *graph.Graph {
 		s.NumUnit = numUnits
 	}
 
+	// Remove tag marking samples from the base profiles, so it does not appear
+	// as a nodelet in the graph view.
+	prof.SetTag("pprof::base", []string{})
```

Comment:
I think this will set the tag on all samples even on those that do not have it, needlessly consuming memory?

Reply:
It would.
Fixed.